_includes/analytics.html

```diff
@@ -1,8 +1,4 @@
-<script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-  ga('create', '{{ site.analytics }}', 'auto');
-  ga('send', 'pageview');
-</script> <!-- End Analytics -->
+<!-- Google Tag Manager (noscript) -->
```

Comment:
Now you must add a total of two scripts, this would be the first one that has to go to the beginning of the body

Reply:
and this would be the second script that we have to place the script as far as possible from the head